### ISSUE TITLE
Feature to Track and Visualize Training/Validation Loss in yolov10

### ISSUE THREAD

**Issue by mabubakarsaleem** [THU-MIG/yolov10#289]
How can we obtain the loss graphs of training and validation in YOLOv10 training?

**Comment by lgarciaeseneye** [comment]
Hi, I'm also interested on it, did you finally managed to solve it?
